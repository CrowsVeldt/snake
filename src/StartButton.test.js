/* eslint-env jest */
it.skip()
