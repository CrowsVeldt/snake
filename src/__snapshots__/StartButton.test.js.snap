// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The StartButton Component renders correctly 1`] = `undefined`;
